appinventor/components/src/com/google/appinventor/components/runtime/Canvas.java

```diff
@@ -993,6 +1011,19 @@ public void BackgroundImage(String path) {
     view.setBackgroundImage(path);
   }
 
+  /**
+   * Specifies the back ground image in Base64 format
+   * imageUrl will be in format of:  data:image/png;base64,iVBORw0KG...s//f+4z/6Z
+   * @param imageUrl the base64 format for an image
+   */
+  @android.support.annotation.RequiresApi(api = android.os.Build.VERSION_CODES.FROYO)
+  @SimpleProperty (
+          description = "Set the back ground image in Base64 format"
+  )
+  public void BackGroundImageinBase64(String imageUrl) {
```

Comment:
Keep the case consistent with the existing properties `BackgroundColor` and `BackgroundImage`.

Reply:
Add a check that if the version code of the current build is less than FROYO that nothing happens, and also note this in the documentation.

Reply:
Addressed both comments.